components/camera/camera_test.go

```diff
@@ -250,3 +251,172 @@ func TestCameraWithProjector(t *testing.T) {
 
 	test.That(t, cam2.Close(context.Background()), test.ShouldBeNil)
 }
+
+// testCamera is a simple camera implementation for testing GetImage and GetImages utility functions.
+type testCamera struct {
```

Comment:
Do your tests test cases when GetImages outputs an image in the viam rgba or depth format and you need to convert that into a PNG or JPEG?

Reply:
Actually depth to png is not tested. I added.